src/style.css

```
@@ -1 +1,13 @@
 /* styles go here */
+
+.inputForm > :not(:nth-child(1)) {
+  margin-top: 20px;
+}
+
+.fieldsetBlock .inputBlock:not(:nth-child(2)) {
+  margin-top: 10px;
+}
```

same here, no need to specify parent block
```suggestion
.inputBlock:not(:nth-child(2)) {
  margin-top: 10px;
}
```